packages/console/src/pages/SignInExperience/components/LanguagesForm.tsx

```diff
@@ -21,8 +24,10 @@ type Props = {
 
 const LanguagesForm = ({ isManageLanguageVisible = false }: Props) => {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
-  const { watch, control, register } = useFormContext<SignInExperienceForm>();
+  const { data: signInExperience } = useSWR<SignInExperience, RequestError>('/api/sign-in-exp');
+  const { watch, control, register, setValue } = useFormContext<SignInExperienceForm>();
   const isAutoDetect = watch('languageInfo.autoDetect');
+  const selectedDefaultLanguage = watch('languageInfo.fallbackLanguage');
```

Comment:
Do we have a task to rename `fallbackLanguage` to `defaultLanguage`?

Reply:
Cool, updated.

Reply:
For compatibility reasons, we don't.